src/improvements/tasks/MultisigTask.sol

```diff
@@ -76,7 +76,7 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
     /// @notice Enum to determine the type of task
     enum TaskType {
         L2TaskBase,
-        SimpleBase
+        SimpleTaskBase
     }
```

Comment:
Should OPCMTaskBase also be in this enum, since it's an option for template creation?

Reply:
I'm not sure why it wasn't added in originally. I'll take a look.